### ISSUE TITLE
What is the utility of filter function in mapDispatchToProps?

### ISSUE THREAD

**Issue by moduv5** [bartonhammond/snowflake#120]
Why are you doing something like this in your mapDispatchToProps functon?
```javascript
const creators = Map()
          .merge(...actions)
          .filter(value => typeof value === 'function')
          .toObject();
```

The function could look like: 

```javascript 
function mapDispatchToProps(dispatch) {
  return {
    actions: bindActionCreators(Object.assign({}, profileActions, globalActions), dispatch),
    dispatch
  };
}
```

**Comment by bartonhammond** [comment]
I did it that way as it was recommended back when I first looked at Redux,
like last year in Dec.  The only things needed are functions.

I'm sure Redux has better ways to reduce the boilerplate and I haven't gone
back to review the current state of design patterns.

Would you like to do a PR and address these?  Otherwise, I probably won't
address it as it's not broken.

**Comment by bartonhammond** [comment]
Fixed w/ PR #123 & PR #124 
Thanks @moduv5 - good work!
